flink-container/docker/README.md

```diff
@@ -13,26 +13,28 @@ Install the most recent stable version of [Docker](https://docs.docker.com/insta
 Images are based on the official Java Alpine (OpenJDK 8) image.
 
 Before building the image, one needs to build the user code jars for the job.
-Assume that the job jar is stored under `<PATH_TO_JOB_JAR>` 
+Assume that the job jar is stored under `<COMMA_SEPARATED_PATH_TO_JOB_ARTIFACTS>`
```

Comment:
Seems not to fit. I guess the sentence needs to be changed to something like: `A Flink job can consist of multiple jars. In order to specify the required jars, they need to be passed to `--job-artifacts` of the build script. The individual paths are comma separated.

Reply:
+1. Only one concern: what about changing `jars` to `artifacts`?